alws/releases.py

```diff
@@ -121,23 +93,26 @@ async def get_release_plan(db: Session, build_ids: typing.List[int],
     result = await db.execute(repo_q)
     prod_repos = [
         {
+            'id': repo.id,
             'name': repo.name,
             'arch': repo.arch,
             'debug': repo.debug
         }
         for repo in result.scalars().all()
     ]
 
+    repos_mapping = {RepoType(repo['name'], repo['arch'], repo['debug']): repo
+                     for repo in prod_repos}
+
     if not settings.package_oracle_enabled:
         return {
             'packages': [{'package': pkg, 'repositories': []}
                          for pkg in pulp_packages],
             'repositories': prod_repos
         }
 
-    oracle_response = await OracleClient(settings.packages_oracle_host).post(
+    oracle_response = await BeholderClient(settings.packages_oracle_host).post(
```

Comment:
If you deleted OracleClient, so there doesn’t any sense to use packages_oracle_host, better to use var for beholder host